sql2.txt

```diff
@@ -2,44 +2,44 @@
 to join the OrderDetails table with the product table.  For TotalCount, you will Use
 SUM(Quantity) AS TotalCount .  Order by product name.
 You will need to group by product ID.  Paste your SQL statement below.
-
+SELECT products.ProductName, SUM(OrderDetails.Quantity) AS TotalCount  FROM OrderDetails JOIN products ON OrderDetails.ProductID = products.ProductID GROUP BY OrderDetails.ProductID ORDER BY ProductName;
 
 2. Retrieve the customer name and OrderDetailsID for all the OrderDetails for each customer,
 ordered by customer name. Paste your SQL statement below. You will have to join the Customers
-table with the Orders table, and the Orders table with the OrderDetails table, 
+table with the Orders table, and the Orders table wOith the OrderDetails table, 
 because the Orders table is the one that ties the Customer table to the OrderDetails table.
-
+SELECT Customers.customerName,  OrderDetails.OrderDetailID FROM Customers JOIN  OrderDetails ON Customers.CustomerID = Orders.CustomerID JOIN Orders ON Orders.OrderID = OrderDetails.OrderID ORDER BY Customers.customerName;
 
 3. Extend the previous query to join the Products table.  You are now joining the Customers, Orders,
 OrderDetails, and Products tables.  The result of your query should include the
 customer name, the product name, and the total quantity of that product ordered.  Note that a customer
 may have ordered the same product in several different orders.  So the total quantity is
 SUM(Quantity) AS TotalQuantity and you must GROUP BY CustomerID, ProductID.  Paste your
 SQL statement below.
-
+SELECT Customers.customerName, products.ProductName, SUM(OrderDetails.Quantity) AS TotalQuantity FROM Customers JOIN Orders ON Customers.CustomerID = Orders.CustomerID JOIN OrderDetails ON Orders.OrderID = OrderDetails.OrderID JOIN products ON OrderDetails.ProductID = products.ProductID GROUP BY Customers.CustomerID, products.ProductID, products.ProductName;
```

Comment:
The question says you must GROUP BY CustomerID, ProductID.. you are not required to add p.ProductName and add ORDER BY CustomersCustomerName, kindly fix it